staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -106,10 +107,16 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 		)
 
 		userInfo, _ := request.UserFrom(ctx)
-		staticAdmissionAttributes := admission.NewAttributesRecord(nil, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Update, userInfo)
 		admissionCheck := func(updatedObject runtime.Object, currentObject runtime.Object) error {
-			if mutatingAdmission, ok := admit.(admission.MutationInterface); ok && admit.Handles(admission.Update) {
-				return mutatingAdmission.Admit(admission.NewAttributesRecord(updatedObject, currentObject, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Update, userInfo))
+			if mutatingAdmission, ok := admit.(admission.MutationInterface); ok {
```

Comment:
We shouldn't change the patcher, because it doesn't currently allow create on update anyway.

Reply:
@jennybuckley Thx for reviewing. I thought that patcher would finally call `store.Update` here. So the object would also be created when its `AllowCreateOnUpdate` is true. Am I missing sth?😃[https://github.com/kubernetes/kubernetes/blob/e49e3baa832986ba57e9d155333a9d49b0e13b32/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go#L359](https://github.com/kubernetes/kubernetes/blob/e49e3baa832986ba57e9d155333a9d49b0e13b32/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go#L359)

Reply:
Thats true, but that the patcher also checks if the object already exists in it's applyPatch function and returns an error if the object doesn't exist yet.

Reply:
@jennybuckley Make sense 👍